### PR TITLE
Doc generation under Windows

### PR DESCRIPTION
**Pull request by skoudoro**
The goal of this PR is to permit the doc generation under windows.

I use the opportunity to fix many warning during the doc generation like:
- Unknow Field Note instead of Notes
- Fix some link
- Fix docstring style

### CODE REVIEW

**Comment by skoudoro** [comment]
Ok, this PR is done and ready to be reviewed!

**Comment by skoudoro** [comment]
Ok, this PR is done and ready to be reviewed!

**Comment by skoudoro** [comment]
Can you restart Appveyor @arokem? Thank you?

File: dipy/reconst/dti.py [hunk 1]
```diff
@@ -720,7 +720,7 @@ def __init__(self, gtab, fit_method="WLS", return_S0_hat=False, *args,
         is advisable to keep an eye on memory consumption as this value is
         increased.
 
-        Example : In :func:`iter_fit_tensor` we have a default step value of
+        e.g.: In :func:`iter_fit_tensor` we have a default step value of
```
Comment (arokem) [inline dipy/reconst/dti.py]: The phrase "e.g." should always be followed by a comma. And in this case, capitalized, as it's in the beginning of a sentence.

Comment (arokem) [inline dipy/reconst/dti.py]:
```suggestion
        E.g., in :func:`iter_fit_tensor` we have a default step value of
```

**Comment by skoudoro** [comment]
> Why was "Example" not a good word there? Does it trigger a sphinx warning?

Yes, it triggers a warning. As soon as it differs from `Numpydoc` standard, we get a warning with Sphinx and or documentation generation can be impacted.

**Comment by arokem** [comment]
OK. Off we go!

**Pull request merged**